src/quark-core/src/QuarkWallet.sol

```diff
@@ -482,31 +482,28 @@ contract QuarkWallet is IERC1271 {
         assembly {
             // TODO: TSTORE the callback slot to 0
```

Comment:
We can address this TODO by also setting the callback slot to 0 after the callcode.

Reply:
Curious if there's a big benefit here, since tstore will clear naturally? I guess all of these clears are mostly in the camp of "if you do nesting, you're 8% safer"?